authors/apps/articles/tests/test_articles.py

```diff
@@ -436,3 +443,47 @@ def test_filter_by_title(self):
                         status.HTTP_200_OK
                         )
 
+    def test_get_article_with_reading_time(self):
+        """ tests if an article is returned with it's reading_time """
+        token = self.create_user(VALID_USER_DATA)
+        response = self.create_article(
+            token=token,
+            article=VALID_ARTICLE
+        )
+        get_article_url = reverse('crud-article', kwargs={
+            'slug': response.data['article']['slug']
+        })
+        response = self.client.get(
+            get_article_url
+        )
+        self.assertIn('less than a minute read', 
```

Comment:
Good work mate. Though Kindly refactor your code to adhere to our TTL's coding conventions. Lines 459, 462, 482, the arguments should be on the next/separate line. Otherwise Great work on the implementation.

Reply:
Thank you @Genza999. Working on it.

Reply:
@Genza999 I have finished refactoring the lines of code in tests